spec/features/tslr_claim_spec.rb

```diff
@@ -29,34 +29,15 @@
 
     expect(page).to have_text("You are eligible to claim back student loan repayments")
 
-    click_on "Skip GOV.UK Verify"
+    perform_verify_step
 
-    expect(page).to have_text(I18n.t("tslr.questions.full_name"))
-
-    fill_in I18n.t("tslr.questions.full_name"), with: "Margaret Honeycutt"
-    click_on "Continue"
-
-    expect(claim.reload.full_name).to eql("Margaret Honeycutt")
-
-    expect(page).to have_text(I18n.t("tslr.questions.address"))
-
-    fill_in_address
-
-    expect(claim.reload.address_line_1).to eql("123 Main Street")
-    expect(claim.address_line_2).to eql("Downtown")
-    expect(claim.address_line_3).to eql("Twin Peaks")
-    expect(claim.address_line_4).to eql("Washington")
-    expect(claim.postcode).to eql("M1 7HL")
-
-    expect(page).to have_text(I18n.t("tslr.questions.date_of_birth"))
-
-    fill_in_date_of_birth
-
-    expect(page).to have_text(I18n.t("tslr.questions.payroll_gender"))
-    choose "Male"
```

Comment:
Yeah, I think you may be right...

Reply:
Can we get this merged and tackle that separately? I just started to add a feature spec for completing missing verify fields but it's going to be involved and it would be good to get this merged today if we can so we're ready for Monday.

Reply:
Sounds good to me 👍